packages/web-components/fast-foundation/src/radio-group/radio-group.template.ts

```diff
@@ -10,10 +10,12 @@ export function radioGroupTemplate<T extends FASTRadioGroup>(): ElementViewTempl
     return html<T>`
         <template
             role="radiogroup"
+            tabindex=${x => (x.disabled ? -1 : void 0)}
             aria-disabled="${x => x.disabled}"
             aria-readonly="${x => x.readOnly}"
             aria-orientation="${x => x.orientation}"
             @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+            @mousedown="${(x, c) => x.handleDisabledClick(c.event as MouseEvent)}"
```

Comment:
It would be great if we could keep a naming convention for top-level events, to separate the event handler from what it's specifically used for from an implementation standpoint. I typically use `<event>Handler`, which makes it easy to find the handler by searching for the exact event name.

(side nit: I'd consider `focusOutHandler` to also be out of convention here, since the event name is specifically `focusout` and not `focusOut` or `focus-out`).

Reply:
Yeah, I went back and forth on this to be honest. I went for a convention which detailed the semantic intent here, but we can certainly look to change. In _most_ cases I agree for these, but this specifically I didn't want to create the perception that there was general utility with it.